documentation/2.0/content/userguide/target_env.md

```diff
@@ -130,6 +132,22 @@ kubernetes:
 ```
 These fields will override the values in the output file, and the file would be rewritten with the revised values. List values in the model will be combined with existing values in the output file. For example, if `my-cluster` was in the original output file, the model content for `my-cluster` would be merged with it, overriding the `replicas` value. If `my-cluster` was not in the original output file, it would be added to the list of clusters.
 
+When creating a resource file for WebLogic Kubernetes Operator 4.0, the `kubernetes` section of the model has to be structured differently. This is because there are multiple documents in the resource file for version 4.0. This example uses `domain` and `cluster` folders that will merge with the corresponding documents in the resource file:
```

Comment:
WebLogic Kubernetes Operator 4.0 => WebLogic Kubernetes Operator 4.0.0+

Reply:
Do we also need to state that the `wko` targets are for WKO 3.x just to be clear?